tripal/tripal.views.inc

```diff
@@ -116,60 +118,91 @@ function tripal_views_data_alter(&$data) {
  * Integreates the Tripal fields with Views.
  */
 function tripal_views_data_fields(&$data) {
-
+  
+  // Get the bundle details so we only look this up once.
+  $all_bundles = [];
+  $sql = "
+    SELECT TB.name, TV.vocabulary, TT.accession
+    FROM {tripal_bundle} TB
+      INNER JOIN {tripal_term} TT on TT.id = TB.term_id
+      INNER JOIN {tripal_vocab} TV on TV.id = TT.vocab_id
+  ";
+  $results = db_query($sql, [':bundle_name' => $bundle_name]);
```

Comment:
The args array shouldn't exist here I think.

Thanks!

Reply:
Yes ,you are right.